main/src/cgeo/geocaching/utils/AngleUtils.java

```diff
@@ -35,7 +35,8 @@ public static float difference(final float from, final float to) {
      * @return the same angle in the [0, 360[ range
      */
     public static float normalize(final float angle) {
-        return (angle >= 0 ? angle : (360 - ((-angle) % 360))) % 360;
+        final float mod = angle % 360;
+        return mod >= 0 ? mod : (mod + 360) % 360;
```

Comment:
Yes, because -Float.MIN_VALUE + 360 becomes 360, which is not within the desired interval.